src/main/kotlin/com/dp/advancedgunnerycontrol/weaponais/AGCUitls.kt

```diff
@@ -90,7 +91,7 @@ fun isOpportuneTarget(tgt : CombatEntityAPI?, predictedLocation: Vector2f?, weap
     if(target.maxSpeed > weapon.projectileSpeed * trackingFactor) return false
     val ttt = (weapon.location - p).length() / weapon.projectileSpeed
     val ammoLessModifier = if(!weapon.usesAmmo()) 1.0f else if (weapon.ammoTracker.reloadSize > 0f) 0.5f else 0.1f
-    if(((p - weapon.location).length() - target.collisionRadius * ammoLessModifier + ttt * target.maxSpeed * 0.1f / ammoLessModifier) >
+    if(((p - weapon.location).length() - effectiveCollRadius(target) * ammoLessModifier + ttt * target.maxSpeed * 0.1f / ammoLessModifier) >
```

Comment:
Some calls to `target.collisionRadius` weren't wrapped in `effectiveCollRadius`. It wasn't intentional, I hope? Anyway, I wrapped all.